frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -486,7 +486,10 @@ export default function SurveyEdit(): JSX.Element {
                                                                 placeholder="ex: .className or #id"
                                                             />
                                                         </LemonField.Pure>
-                                                        <LemonField.Pure label="Survey wait period">
+                                                        <LemonField.Pure
+                                                            label="Survey wait period"
+                                                            info="Note that this condition will only apply to one browser for a given non-anonymous user.  If the user switches browsers or uses an incognito session, they could see this survey again."
+                                                        >
```

Comment:
I kinda just fired from the hip on this copy, open to suggestions if folks have any.

Reply:
this is a bug 😅 , @Phanatic we found yet another candidate for the `internal_targeting_flag` so this works cross-browser